nuxeo-drive-core/src/main/java/org/nuxeo/drive/service/impl/DefaultFileSystemItemFactory.java

```diff
@@ -123,8 +125,18 @@ public boolean isFileSystemItem(DocumentModel doc, boolean includeDeleted, boole
             }
             return false;
         }
+        // Try to fetch blob
+        Blob blob = null;
+        try {
+            blob = getBlob(doc);
+        } catch (NuxeoException e) {
```

Comment:
If in real life you can catch something more specific than `NuxeoException`, like `PropertyException`, it would be better (of course make your test `ExceptionBlobHolder` throw that).